src/app/views/query-response/adaptive-cards/AdaptiveCard.tsx

```diff
@@ -99,22 +101,57 @@ class AdaptiveCard extends Component<IAdaptiveCardProps> {
       this.adaptiveCard.parse(data);
       const renderedCard = this.adaptiveCard.render();
       return (
-        <div
-          style={{
-            minHeight: '500px',
-            maxHeight: '800px',
-            overflowY: 'auto',
-          }}
-          ref={(n) => {
-            if (n && !n.firstChild) {
-              n.appendChild(renderedCard);
-            } else {
-              if (n && n.firstChild) {
-                n.replaceChild(renderedCard, n.firstChild);
-              }
-            }
-          }}
-        />
+        <Pivot className='pivot-response'>
+          <PivotItem
+            key='card'
+            ariaLabel={translateMessage('card')}
+            headerText={'Card'}
+            style={{
+              minHeight: '500px',
+              maxHeight: '800px',
+              overflowY: 'auto',
+            }}
+          >
+            <div
+              ref={(n) => {
+                if (n && !n.firstChild) {
+                  n.appendChild(renderedCard);
+                } else {
+                  if (n && n.firstChild) {
+                    n.replaceChild(renderedCard, n.firstChild);
+                  }
+                }
+              }}
+            />
+          </PivotItem>
+          <PivotItem
+            key='templateJSON'
+            ariaLabel={translateMessage('Template JSON')}
+            headerText={'Template JSON'}
```

Comment:
Maybe this should read the other way. "JSON template"

Reply:
Just opted to go with JSON Schema as you had suggested before